packages/aws-cdk/lib/api/deploy-stack.ts

```diff
@@ -149,13 +149,17 @@ export async function deployStack(options: DeployStackOptions): Promise<DeploySt
   const deployName = options.deployName || stackArtifact.stackName;
```

Comment:
@shivlaks do you mind giving this a quick review?

Reply:
on it